.github/workflows/publish-commit-bottles.yml

```
@@ -87,6 +90,12 @@ jobs:
           GIT_COMMITTER_EMAIL: 1589480+BrewTestBot@users.noreply.github.com
           HOMEBREW_GPG_PASSPHRASE: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY_PASSPHRASE }}
 
+      - name: Enable PR automerge
+        run: gh pr merge --auto ${{github.event.inputs.pull_request}}
+
+      - name: Add CI-published-commit-bottles label
+        run: gh pr merge --add-label CI-published-commit-bottles ${{github.event.inputs.pull_request}}
```

```suggestion
        run: gh pr edit --add-label CI-published-commit-bottles ${{github.event.inputs.pull_request}}
```

https://cli.github.com/manual/gh_pr_edit